pp_netlib/gt_prune.py

```diff
@@ -0,0 +1,293 @@
+import sys
+import operator
+import numpy as np
+from scipy.stats import rankdata
+from pp_netlib.utils import gen_unword, read_isolate_type_from_csv, print_external_clusters
+import graph_tool.all as gt
+from collections import defaultdict, Counter
+
```

Comment:
The functions in this file all need at least a comment describing what they do (or even a full docstring, if appropriate)